cedar-policy-validator/src/schema.rs

```diff
@@ -1013,7 +1019,7 @@ impl ValidatorSchema {
             for p_entity in action.applies_to.applicable_principal_types() {
                 match p_entity {
                     EntityType::Concrete(p_entity) => {
-                        if !entity_types.contains_key(p_entity) {
+                        if !entity_types.contains_key(&p_entity) {
```

Comment:
No `&` here (and below). vscode's rust-analyzer and clippy disagree on this, and this line has churned a couple times already